.goreleaser-stable.yaml

```diff
@@ -57,20 +57,23 @@ nix:
       cp -vr ./dist/kraft $out/bin/kraft
 
 builds:
+#@ targets = {"linuxu": {"os": "linux", "arch": "amd64"}, "darwin-arm64": {"os": "darwin", "arch": "arm64"}, "darwin-amd64": {"os": "darwin", "arch": "amd64"}}
```

Comment:
I would turn this simply into tuples, it's also possible to use multiple lines using ytt:
```yaml
#@ targets = [ \
#@   ["linux", "amd64"], \
#@   ["linux", "arm64"], \
#@ ]
```
Should make git blames easier to read

Reply:
Thanks. It's a dictionary so that the name of the output directory stays the same (kraft-linuxu_linux_amd64_v1)